build.gradle

```diff
@@ -50,11 +57,14 @@ dependencies {
 
     // Needed for build and run
     implementation "org.slf4j:slf4j-api:${slf4jVersion}"
+    implementation "org.apache.httpcomponents:httpclient:${httpclientVersion}"
+    implementation "org.odpi.egeria:topic-integrator-api:${egeriaversion}"
+    implementation 'com.google.code.gson:gson:2.9.0'
```

Comment:
2.10.1 is now current -- but good idea to add as a variable, also if in the egeria pom they can be skipped as above